colossus-tests/src/test/scala/colossus/protocols/http/client/StreamingHttpResponseParserSpec.scala

```diff
@@ -109,21 +138,23 @@ class StreamingHttpResponseParserSpec extends WordSpec with MustMatchers with Tr
       }
 
     }
-    "decode a response that was encoded by colossus with no body" in {
+    "decode a response that was encoded by colossus with no body" ignore {
       validateEncodedStreamingHttpResponse(ByteString(""))
     }
 
-    "decode a response that was encoded by colossus with a body" in {
```

Comment:
At least one of them depends on #137 .  Once these PR's get merged in I'm going to make a pass and try to address all of the ignored tests.